teams/teams-ps/teams/New-CsTeamsAIPolicy.md

```diff
@@ -82,6 +82,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Description
+Enables administrators to provide explanatory text about the Teams AI policy.
```

Comment:
What are the possible values for this? Are there any limitations?

Reply:
@michbrown-png thanks for the review - bt:Description is the type that is used. This is used by almost all policies right now, and I borrowed the wording from their documentation - it should be any string